main/authorization.bs

```diff
@@ -19,3 +21,31 @@ the HTTP `Link` header with a `rel` value of `acl` [[#acl]].
 In the event that a server can't apply an ACL to a resource, it MUST deny
 access.
 [[Source](https://github.com/solid/specification/issues/130#issue-532777017)]
+
+Servers exposing client's access privileges on a resource URL MUST advertise
+by including the `WAC-Allow` HTTP header in the response of HTTP `HEAD` and
+`GET` requests.
+
+The syntax for `WAC-Allow`, using the ABNF syntax is:
+
+```
+WAC-Allow           = "WAC-Allow" ":" OWS permissions OWS
+permissions         = user-permissions OWS "," OWS public-permissions
```

Comment:
The syntax we had in mind is actually more generic, as you can see from the implementation: https://github.com/solid/wac-allow/blob/v1.0.0/src/WacAllowParser.js#L24

The `permissions` block is a comma-separated, unordered list of an arbitrary number of `permission` blocks. A `permission` block is `group = quoted-access-modes`, where `group` is a string of word characters and arbitrary whitespace is allowed around the equals sign.
It just so happens there are two pre-defined groups `user` and `public`.

See some of the tests as well: https://github.com/solid/wac-allow/blob/v1.0.0/__tests__/WacAllowParser-test.js

Reply:
The initial PR was based off the draft spec, not the implementation. Updated ABNF: https://github.com/solid/specification/commit/6d1bc9e1a90f63294e6bc38e6645665e6904121f . If I'm not mistaken, it can parse inputs like:
`WAC-Allow: ,user=" READ  wriTe   Append control ", public=" read append" ,other="read " , group="  ",,  `